### ISSUE TITLE
Clarify how we plan to do the train/test split and do validation

### ISSUE THREAD

**Issue by wbwakeman** [AllenInstitute/DeepCell#20]
Need to plan how we do the train/test split on the labeled data.  Plan and document the options.  Present to Pika and scientific group for discussion.
Record agreed-upon strategy.

Notes:
Can split by ROI or REGION or EXPERIMENT
Scientists will give us a list experiments to include in test set
train/test split on remaining experiments is random
Each experiment is in the test or the training set

Tasks
- [x] Pika will develop a proposal for how we think test/train/validation splitting should be done when training the classifier
- [ ] Write up slides/create some kind of forma proposal
- [x] Meet with stakeholders to propose plan and solicit feedback (minimum need Michael Buice, Jerome Lecoq, and Jun Zhuang to make decision)

**Comment by danielsf** [comment]
This ticket is a conversation with the scientists. Might want to wait until they have seen the data, as opinions may solidify during the labeling effort.
